cmd/config/config.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/lyft/flytestdlib/config"
```

Comment:
Can we migrate all of them as well
```suggestion
	"github.com/flyteorg/flytestdlib/config"
```

And to make it work without actually changing the upstream you can in `go.mod` add a replace directive

replace (
github.com/flyteorg/flytestdlib => github.com/lyft/flytestdlib/config
)

And do the same for all

Reply:
Reading up on --> https://golang.org/ref/mod#go-mod-file-replace 

I have no experience with replace directive.  But, I had imagined updating ALL of these (without replace directive).  Started with this commit.  But, the others are super easy to follow.  

Do you have a sense of pros/cons, why the replace directive is better (or is it supposedly less typing, since only changing in one place)?  

If we want the same for also flytestdlib, flyteidl as well, easy enough to do.  Are there others?  Or that would cover it?

Reply:
Ah, I think (if not using go.mod redirect), we need to update each module/repo's self-references.  Then once committed, we should be able to refer out to them (at least is how I am inferring the unit test fail once I tried to make the suggested changes).  It is late, so I could also not be thinking straight ;-)